portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -75,7 +69,7 @@ private String getRandomUsername() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int requestedNumInt = Integer.parseInt(request.getParameter("commentRequestedNum"));
```

Comment:
Oh, looks like just one more field that could be pulled out for consistency with the others.  Haha.. this one will be a bit long.  COMMENT_REQUESTED_COUNT_ENTITY_PROPERTY?